master/internal/api_task.go

```diff
@@ -30,3 +31,25 @@ func (a *apiServer) GetTask(
 			errors.Wrapf(err, "error fetching task %s from database", req.TaskId)
 	}
 }
+
+func (a *apiServer) GetTaskConfig(
+	ctx context.Context, req *apiv1.GetTaskConfigRequest,
+) (resp *apiv1.GetTaskConfigResponse, err error) {
+	if err := a.canDoActionsOnTask(ctx, model.TaskID(req.TaskId),
+		expauth.AuthZProvider.Get().CanGetExperimentArtifacts); err != nil {
+		return nil, err
+	}
+
+	t := &taskv1.Task{}
```

Comment:
nit https://github.com/uber-go/guide/blob/master/style.md#use-var-for-zero-value-structs